src/Symfony/Component/HttpKernel/Tests/HttpKernelTest.php

```diff
@@ -28,6 +28,7 @@
 use Symfony\Component\HttpKernel\HttpKernel;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 use Symfony\Component\HttpKernel\KernelEvents;
+use Throwable;
```

Comment:
We're not using `use` statement for built-in classes, use `\Throwable` in the code.

Reply:
Done.